shared/types/api/wallet.go

```diff
@@ -23,6 +23,9 @@ type WalletStatusResponse struct {
 	PasswordSet       bool           `json:"passwordSet"`
 	WalletInitialized bool           `json:"walletInitialized"`
 	AccountAddress    common.Address `json:"accountAddress"`
+	NodeAddress       common.Address `json:"nodeAddress"`
+	HasAddress        bool           `json:"hasAddress"`
```

Comment:
What's the use for HasAddress?

Reply:
Ah this is something I forgot to clean up. Thanks for catching it.